_data/contacts.yml

```diff
@@ -381,4 +381,8 @@ housing_unit_inventory:
     email: jdadams@utah.gov
   - name: the analytics team
     agency: WFRC
-    email: analytics@wfrc.org
\ No newline at end of file
+    email: analytics@wfrc.org
+jacob:
```

Comment:
**chore**: please change this to be more generic. For example `h3_indices` or something like that.

Reply:
What's the desired benefit of the contacts page? It seems like I could have half a dozen different entries in contacts.yml for myself, all repeating the same info. Since the h3 contact very likely won't get reused anywhere else, what's the point of having a one-off contact instead of a generic one for myself that I can reuse in other pages?

Reply:
Because when you win the lottery and leave us behind, we'll have to replace your name in all of those locations instead of divvy up the "products" you were getting feedback on. Ideally we don't have too many one offs but this is very specific.